scripts/generate

```diff
@@ -321,8 +390,11 @@ def('toc',
             const level$ = Number(hN[1]);
             const level$$ = level$ > level ? hN === tagName ? level : level + 1 : level$;
 
+            const pattern = '<a [^>]*>([^<]*)</a>';
             const innerHtml =
-            replace(/^(<code><a [^>]*>)([^ ]*)/, '$1<b>$2</b>', _innerHtml);
+            pipe([replace(regex('', pattern), '<b>$1</b>'),
+                  replace(regex('g', pattern), '$1')],
+                 _innerHtml);
```

Comment:
The first `<a href="...">...</a>` we encounter is the function name, which should appear in bold in the table of contents. The remaining ones should simply be unlinked.